src/components/styles/LandingPage.sass

```diff
@@ -0,0 +1,70 @@
+$button-width: 13rem
+
+.landing-content
+    display: flex
+    flex-direction: column
+    align-items: center
+    >*
+        flex-grow: 0
+        flex-shrink: 0
+
+#info
+    // margin-left: 10%
+    width: 80%
+    max-width: calc(40em + #{$button-width} + 10em)
+    display: inline-block
+    width: 100%
+    #info-title
+        border-bottom: 1px solid #555
+    #info-content
+        [role="button"]
+            width: 100%
+            height: 100%
+        >*
+            box-sizing: border-box
+        #demo-sponsor
+            box-sizing: border-box
+            padding: 1em
+            >*
+                height: 4em
+                max-width: 100%
+                margin-bottom: 1em
+        p
+            text-indent: 1em
+            padding-bottom: 1em
+            box-sizing: border-box
+            display: inline-block
+
+.slick-slide img
+    margin: auto
+    height: 10em
+    // width: 10em
+
+// .landing-carousel
+//     flex-basis: 10em
+//     width: 10%
+//     // flex: 1
+//     display: inline-block
+//     background-color: #333
+//     .slick-list
+//         height: 100%
+
+// .carousel-image
+//     // height: 100%
+//     // width: 100%
+//     // display: flex
+//     // justify-content: center
+//     // align-items: center
+//     img
+//         // flex: 0
+//         max-width: 100%
+//         max-height: 100%
+//         display: inline-block
```

Comment:
```suggestion

```
Not sure if we'd need these css classes/comments at all in the future but if you don't think we would then it'd probably be best to remove them